integrationTests/tests/redis/redisCachingDriver.test.js

```diff
@@ -1,27 +1,21 @@
 /*
- * Copyright (c) 2019, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
  * All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 "use strict";
-import chai from "chai";
-import chaiAsPromised from "chai-as-promised";
 
-import delayedTests from "../test/cache/delayedTests.tests";
```

Comment:
Why do this instead of just directly invoking the tests since this is the only driver for those tests?

Reply:
This is not the only driver

Reply:
Where is the other one?

Reply:
/redis folder